Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -0,0 +1,243 @@
+---
+versionFrom: 8.0.0
+---
+
+# Quick start
+
+_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 3 internal indexes which should not be used for searching when returning results on a public website because it indexes content that has not been published yet. It also ships with an external index that you can use to get up and running._
+
+## Performing a search
+
+:::note
+In the coming examples the Umbraco Starter Kit has been used, this is because it gives some content to be searched upon quickly. Some of the examples may require setting up templates, etc if you are following on your own site.
+:::
+
+The starter kit comes with some templates and doc types and content nodes set up already, so let's just use some of those to set up a simple search system.
+
+Let's make it possible to search on the /people page.
+
+We do this by adding a search bar on the template page, so in the people.cshtml template we can add this at the top, under the nav element:
+
+```csharp
+...
+</nav>
+-->
+
+<div>
+    <form action="@Model.Url" method="get">
+        <input type="text" placeholder="Search" id="query" name="query" />
+        <button>Search</button>
+    </form>
+</div>  
+
+<div class="employee-grid">
+...
+```
+
+This will make an input field at the top of the page and make it redirect to the same page when submitted. Now let's set up the search query using Examine.
```

Comment:
```suggestion
This will create an input field at the top of the page and make it redirect to the same page when submitted. 

Now let's set up the search query using Examine.
```